jdk-1.6-parent/select2-parent/select2/src/main/java/org/wicketstuff/select2/AbstractSelect2Choice.java

```diff
@@ -54,9 +55,11 @@
 	private List<T> choices;
 	private ChoiceRenderer<T> renderer;
 
+	private transient boolean convertValuePerformed = false;

```

Comment:
The name of the field probably should be `convertInputPerformed` to be in sync with the name of the method.